indico/htdocs/sass/partials/_boxes.scss

```diff
@@ -64,17 +64,17 @@
     background-color: lighten($dark-gray, $color-variation);
     cursor: move;
 
-    &:before,
-    & > .handle:before {
+    &::before,
+    & > .handle::before {
```

Comment:
Things like `::` can be enforced by `sass-lint` so we don't need to add them in the wiki. Let's keep it only for things that can be checked programmatically.